modules/navier_stokes/src/loops/GatherRCDataElementThread.C

```diff
@@ -48,7 +49,14 @@ GatherRCDataElementThread::subdomainChanged()
     copied_queries.template condition<AttribVar>(static_cast<int>(var_num)).queryInto(var_eks);
     for (auto var_ek : var_eks)
       if (auto insfv_ek = dynamic_cast<INSFVMomentumResidualObject *>(var_ek))
+      {
+        // On INITIAL, the time integrator is not ready for these residual objects
+        if (this->_fe_problem.getCurrentExecuteOnFlag() == EXEC_INITIAL)
+          if (dynamic_cast<FVTimeKernel *>(var_ek))
+            continue;
```

Comment:
This is where I wonder what the value of executing the VFR on initial is. This will lead to computation of a different velocity than what is used in the first residual evaluation

Reply:
It wont be that different and it will only matter for internal sidesets.
VFR on external boundaries on initial seems important. 

Do we have a way to know if a sideset is internal or on the boundary of the flow domain? Since this issue is for RC coefficients, it's a problem inside the flow domain not on its edge. We could disallow only inside